<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>removeCustomerWidget</class>
 <widget class="QWidget" name="removeCustomerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Customer</string>
  </property>
  <widget class="QPushButton" name="deleteCustomerButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Customer</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete Customer</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>461</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="accountInformationLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Account Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Select...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BC Tree Fruit Inc.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SunRype Inc.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tantalus Winery Inc.</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Account ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="removeCustomerId">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Street Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="removeCustomerStreetAddressLine1">
       <property name="toolTip">
        <string>Street Address</string>
       </property>
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Street Address Line 2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="removeCustomerStreetAddressLine2">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="removeCustomerCity">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="removeCustomerRegion">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Postal Code</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="removeCustomerPostalCode">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="removeCustomerCountry">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="removeCustomerPhoneNumber">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="removeCustomerEmail">
       <property name="placeholderText">
        <string>Auto-fills from Account Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
